node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -82,3 +82,12 @@
         height: 200px;
     }
 }
+
```

Comment:
Can you add a note here that indicate that these are `video.js` overrides and why they're necessary ?